### ISSUE TITLE
Тех задание

### ISSUE THREAD

**Issue by Anton-15** [Anton-15/ARM-DET-SAD#1]

**Comment by Anton-15** [comment]
[ТЗ АРМ дет сад .docx](https://github.com/Anton-15/-./files/5214528/default.docx)
поправил пункт про защиту информации 
Требование к защите информации от несанкционированного доступа
1) должна использоваться концепция работы с Системой только зарегистрированных пользователей, исключающая возможность несанкционированного доступа;
2) каждый пользователь получает доступ в Систему только с использованием пароля.

**Comment by Anton-15** [comment]
Владимир Юрьевич Вы смотрели поправки ?

**Comment by wldmr-key** [comment]
да, хорошо, только слово "исключающая" правильнее заменить на "ограничивающая"

**Comment by Anton-15** [comment]
[ТЗ АРМ дет сад .docx](https://github.com/Anton-15/ARM-DET-SAD/files/5312590/default
Произошли небольшие изменения в связи с изменением среды разработки.

**Comment by Anton-15** [comment]
[ТЗ АРМ дет сад .docx](https://github.com/Anton-15/ARM-DET-SAD/files/5312601/default.docx)
Произошли небольшие изменения в связи с изменением среды разработки.

**Comment by wldmr-key** [comment]
думаю, не последние ))
